app/helper/error.go

```diff
@@ -44,12 +44,12 @@ func StandardErrorMessage(err error) (msg string, ok bool) {
 			if server, err := url.Parse(t.URL); err == nil {
 				host = server.Host
 			}
-			msg = fmt.Sprintf("The connection to the server %s was refused - did you specify the right host or port?", host)
+			msg = fmt.Sprintf("The connection to the server %s was refused - did you specify the right host or port? \n", host)
```

Comment:
Why add a new line here? We usually decide if print a new line or not when we print it instead of put the new line into a message.